docs/configuration/metrics/index.md

```diff
@@ -41,18 +53,23 @@ metrics:
 - `metricDefaults.aggregation.interval` - The default interval which defines over what period measurements of a metric should be aggregated.
 
 ## Metrics
+
 Every metric that is being declared needs to define the following fields:
+
 - `name` - Name of the metric that will be exposed in the scrape endpoint for Prometheus
 - `description` - Description for the metric that will be exposed in the scrape endpoint for Prometheus
 - `resourceType` - Defines what type of resource needs to be queried.
+- `resourceGroupName` - (Optional)  This field is used to override the resource group declared in `azureMetadata`. This allows Promitor to use multiple resource groups in one single yaml file. Provide one resource group per metric.
```

Comment:
Personally find it a bit too technical - How about this?

```suggestion
- `resourceGroupName` - _(Optional)_  Capability to specify a resource group to scrape for this metric. This allows you to specify a different resource group from the one configured in `azureMetadata`. This allows you to scrape multiple resource groups with one single configuration.
```

I left out `Provide one resource group per metric.` as I'm not sure what you mean here, that you need to configure it at least on one place either metric or azure metadata? Isn't this already covered given the one in azure metadata is not optional?

Thoughts?

Reply:
@brusMX 	can you redo this one as well please?